arlas-core/src/main/java/io/arlas/server/utils/UriInfoUtil.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.utils;
+
+import javax.ws.rs.core.UriInfo;
+
+public class UriInfoUtil {
```

Comment:
I personally find the name may be misleading, I expect static methods from an Util class, whereas this is a wrapper.

Reply:
You're right. Would it make more sense to rename the class to UriInfoWrapper or reimplement it with static methods?

Reply:
I don't know how it is used, I feel like there will be more boilerplate code with static methods, but this is your choice :-)

Reply:
Ok. Renamed to UriInfoWrapper